libbeat/autodiscover/providers/kubernetes/kubernetes.go

```diff
@@ -55,11 +55,10 @@ type Provider struct {
 	appenders autodiscover.Appenders
 	logger    *logp.Logger
 	eventer   Eventer
-	keystore  keystore.Keystore
 }
 
 // AutodiscoverBuilder builds and returns an autodiscover provider
-func AutodiscoverBuilder(bus bus.Bus, uuid uuid.UUID, c *common.Config, keystore keystore.Keystore) (autodiscover.Provider, error) {
+func AutodiscoverBuilder(bus bus.Bus, uuid uuid.UUID, c *common.Config, keyStore keystore.Keystore) (autodiscover.Provider, error) {
```

Comment:
nit: we use `keystore` thrughout the code, plus the type is also named `Keystore`. Let's not rename the variable to `keyStore`.

Reply:
👍